docs/guides/dag-construction.md

```diff
@@ -114,6 +119,7 @@ Being part of an entry group does not imply that all vertices in the group get e
 {% endhint %}
 
 Practically speaking, this means that you'll need to add all vertices to the entry group that either (non-exclusive):
+
 - will immediately start execution
 - need to be provided client input for entry ports
 
```

Comment:
note to self: add `outputs` docs

Reply:
https://github.com/Talus-Network/nexus-sdk/pull/184/commits/2beb695980583be9749191b6c0bce172d1720902